<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditToolsClass</class>
 <widget class="QWidget" name="EditToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>56</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditTools</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#EditToolsClass{border:1px solid grey;border-radius:4px;}
                    QWidget{background-color: rgb(235,229,209);}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFontComboBox" name="fontComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"> QComboBox {
     border: 1px solid #A3D9F2;
     border-radius: 3px;
     padding: 1px 18px 1px 3px;
     min-width: 4em;
 }

 QComboBox:editable {
     background: white;
 }

 QComboBox:!editable, QComboBox::drop-down:editable {
      background: rgb(235,229,209);
 }

 /* QComboBox gets the &quot;on&quot; state when the popup is open */
 QComboBox:!editable:on, QComboBox::drop-down:editable:on {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                 stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
 }

 QComboBox:on { /* shift the text when the popup opens */
     padding-top: 3px;
     padding-left: 4px;
 }

 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;

     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"> QComboBox {
     border: 1px solid #A3D9F2;
     border-radius: 3px;
     padding: 1px 18px 1px 3px;
     min-width: 4em;
 }

 QComboBox:editable {
     background: white;
 }

 QComboBox:!editable, QComboBox::drop-down:editable {
      background: rgb(235,229,209);
 }

 /* QComboBox gets the &quot;on&quot; state when the popup is open */
 QComboBox:!editable:on, QComboBox::drop-down:editable:on {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                 stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
 }

 QComboBox:on { /* shift the text when the popup opens */
     padding-top: 3px;
     padding-left: 4px;
 }

 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;

     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="font_up_btn">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:0px groove white;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:0px groove white;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="LxMemo.qrc">
         <normaloff>:/LxMemo/icons/font_up.png</normaloff>:/LxMemo/icons/font_up.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="font_down_btn">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:0px groove white;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:0px groove white;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="LxMemo.qrc">
         <normaloff>:/LxMemo/icons/font_down.png</normaloff>:/LxMemo/icons/font_down.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="font_bold_btn">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:0px groove white;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:0px groove white;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="LxMemo.qrc">
         <normaloff>:/LxMemo/icons/bold.png</normaloff>:/LxMemo/icons/bold.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="font_italic_btn">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:0px groove white;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:0px groove white;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="LxMemo.qrc">
         <normaloff>:/LxMemo/icons/italic.png</normaloff>:/LxMemo/icons/italic.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="font_underline_btn">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:0px groove white;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:0px groove white;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="LxMemo.qrc">
         <normaloff>:/LxMemo/icons/underline.png</normaloff>:/LxMemo/icons/underline.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="font_strikeout_btn">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:0px groove white;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:0px groove white;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="LxMemo.qrc">
         <normaloff>:/LxMemo/icons/strikethrough.png</normaloff>:/LxMemo/icons/strikethrough.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorButton" name="font_color_btn">
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:1px groove #dcdcdc;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:1px groove  #dcdcdc;
}

/* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 0px groove gray;
     border-top-right-radius:1px;
     border-bottom-right-radius: 1px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 12px;
 }

 QToolButton::menu-arrow {
     image: url(:/LxMemo/icons/down-arrow.png);
	width:12px;
	height:12px;
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }
QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorButton" name="font_bkcolor_btn">
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border-radius: 4px;
}
QToolButton:hover {
    background-color:white;
	border:1px groove #dcdcdc;
}
QToolButton:pressed {
	background-color: #e4fffb;
	border:1px groove  #dcdcdc;
}

/* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 0px groove gray;
     border-top-right-radius:1px;
     border-bottom-right-radius: 1px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 12px;
 }

 QToolButton::menu-arrow {
     image: url(:/LxMemo/icons/down-arrow.png);
	width:12px;
	height:12px;
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }
QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="LxMemo.qrc"/>
 </resources>
 <connections/>
</ui>
